conda-recipe/meta.yaml

```diff
@@ -14,96 +14,103 @@
 # limitations under the License.
 #===============================================================================
 
-{% set version = '2024.1' %}
+# NB: this recipe should be synced between sklearnex repo and feedstocks
+
+{% set name = "scikit-learn-intelex" %}
+{% set version = "2025.1" %}
 {% set buildnumber = 0 %}
 
 package:
-    name: daal4py
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
+# Reminder: source should point to path for repo or github archive for feedstock
+# source:
+#   url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
+#   sha256: ...
 source:
-    path: ..
+  path: ..
 
 build:
-    number: {{ buildnumber }}
-    include_recipe: False
-    script_env:
-    - DPCPPROOT
-    - DALROOT
-    ignore_run_exports:
-       - python
-       - mpich
+  skip: true  # [not (linux64 or win)]
+  number: {{ buildnumber }}
+  include_recipe: False
+  script_env:
+  - DPCPPROOT
+  - DALROOT
+  - NO_DIST=1  # [win]
 
 requirements:
-    build:
-        - {{ compiler('cxx') }}  # [not osx]
-        - {{ compiler('c') }}  # [not osx]
-    host:
-        - python
-        - setuptools
-        - numpy {{ numpy }}
-        - dal-devel ==2024.1.0
-        - cython
-        - jinja2
-        - mpich  # [osx]
-        - impi-devel  # [not osx]
-        - clang-tools
-        - pybind11
-        - make
-    run:
-        - python
-        - dpcpp_cpp_rt ==2024.1.0  # [linux]
-        - dpcpp_cpp_rt ==2024.1.0  # [win or osx]
-        - dal ==2024.1.0
-    ignore_run_exports:
-        - numpy
+  build:
+    - make  # [linux]
+    - dpcpp_linux-64  # [linux64]
+    # - dpcpp_win-64  # [win]
+    - {{ compiler('cxx') }}  # [linux64 or win]
+    # conda-forge feedstock specific
+    # - {{ stdlib("c") }}  # [linux64 or win]
+  host:
+    - python
+    - setuptools
+    - cmake
+    - clang-format
+    - cython
+    - jinja2
+    - pybind11
+    - numpy {{ numpy }}
+    - impi-devel  # [not win]
+    # dal-devel pinning depends on the recipe location (repo or feedstock)
+    - dal-devel
+    # - dal-devel =={{ version }}
```

Comment:
Maybe for the current iteration it is fine, but for a such project like `scikit-learn-intelex` I expect that it will be enhanced by using `pyproject.toml`, where requirements defined in one place. It will require to update `pyproject.toml` file, `[build-system]` table is recommended https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#writing-your-pyproject-toml